src/aics-image-viewer/components/ChannelsWidget/index.tsx

```diff
@@ -160,10 +151,7 @@ export default class ChannelsWidget extends React.Component<ChannelsWidgetProps,
                       channelControlPoints={thisChannelSettings[LUT_CONTROL_POINTS]}
                       colorizeEnabled={thisChannelSettings[COLORIZE_ENABLED]}
                       colorizeAlpha={thisChannelSettings[COLORIZE_ALPHA]}
-                      isovalue={thisChannelSettings.isovalue}
-                      opacity={thisChannelSettings.opacity}
```

Comment:
why are these removed?

Reply:
I believe TypeScript's analysis revealed that they were unused in that component. The channel controls set these values via methods, and the only way they're displayed is via the slider tooltip, which comes right out of that component's state.